src/Mobs/Monster.cpp

```diff
@@ -95,12 +95,16 @@ cMonster::cMonster(const AString & a_ConfigName, eMonsterType a_MobType, const A
 	, m_DropChanceBoots(0.085f)
 	, m_CanPickUpLoot(true)
 	, m_BurnsInDaylight(false)
-	, m_RelativeWalkSpeed(1.0)
+	, m_RelativeWalkSpeed(0.5)
```

Comment:
it's called **relative**, so 1 means "the default". If you want a different default, change the base then.